src/eduid_webapp/phone/views.py

```diff
@@ -255,4 +255,21 @@ def resend_code(user, number):
     current_app.stats.count(name='mobile_resend_code', value=1)
 
     phones = {'phones': user.phone_numbers.to_list_of_dicts(), 'message': 'phones.code-sent'}
-    return phones
+    return PhoneListPayload().dump(phones)
```

Comment:
same here, MarshalWith should handle that

Reply:
Don't know what I was thinking...

Reply:
Aw right, It was merging the upgrade of marshmallow